sphinx/config.py

```diff
@@ -169,7 +169,9 @@ class Config(object):
         epub_theme = ('epub', 'html'),
         epub_theme_options = ({}, 'html'),
         epub_title = (lambda self: self.html_title, 'html'),
+        epub_description = ('', 'epub3', [str]),
```

Comment:
Sorry, I forgot to fix epub3.py.
I just fixed it.

As you said, some tests should be added.
I'll add some tests later.